src/components/shareable-url-modal/shareable-url-modal.js

```diff
@@ -35,8 +37,9 @@ const ShareableUrlModal = ({ onToggleModal, visible }) => {
   const [inputValues, setInputValues] = useState({});
   const [isFormDirty, setIsFormDirty] = useState({
     /* eslint-disable camelcase */
-    bucket_name: false,
-    region: false,
+    has_bucket_name: false,
+    has_platform: false,
+    has_endpoint: false,
```

Comment:
my bad, I was giving an example but if we can use camelCase we should do that like (hasBucketName)